README.rst

```diff
@@ -22,30 +27,41 @@ For now, this repo is not intergrated with devstack. You'll be running the app l
 
       mkdir -p ~/workspace/
       cd ~/workspace/
-      git clone https://github.com/edx/devstack.git
+      git clone https://github.com/edx/frontend-app-communications.git
 
 2. Install frontend dependencies
 
    .. code-block::
 
       npm i
 
-3. start the devserver. The app will be running at `localhost:8585`, or whatever port you change it too.
+3. start the devserver. The app will be running at ``localhost:1984``, or whatever port you change it too.
```

Comment:
where does this magic number come from?

Reply:
I made it up! it lives here: https://github.com/edx/frontend-app-communications/blob/8d1cc534b7b3a53841e417b2e4441eb4bc20031d/.env.development#L2